apps/web/components/Form/Form.tsx

```diff
@@ -12,22 +12,30 @@ import {
   Checkbox,
   Button,
   Option,
+  InputFileUpload,
+  UploadFile,
 } from '@island.is/island-ui/core'
 import {
   Form as FormType,
   GenericFormMutation,
   GenericFormMutationVariables,
+  Mutation,
+  MutationCreateUploadUrlArgs,
+  PresignedPost,
 } from '@island.is/web/graphql/schema'
+import { CREATE_UPLOAD_URL } from '@island.is/application/graphql'
```

Comment:
I've stopped using the CREATE_UPLOAD_URL query from island.is/application/graphql and have defined it locally :)

Reply:
I agree that this logic is redundant and could be centralised within a single reusable component and could be shared between applications but I feel like that's something that could be done in another PR alongside the fact that file upload starts as soon as the file gets picked from the users computer rather than once he submits it.